src/coreclr/src/utilcode/util.cpp

```diff
@@ -1007,10 +1009,11 @@ DWORD LCM(DWORD u, DWORD v)
     CPUGroupInfo::GetThreadGroupAffinity(GetCurrentThread(), &groupAffinity);
     m_initialGroup = groupAffinity.Group;
 
-	// only enable CPU groups if more than one group exists
-	BOOL hasMultipleGroups = m_nGroups > 1;
-	m_enableGCCPUGroups = enableGCCPUGroups && hasMultipleGroups;
-	m_threadUseAllCpuGroups = threadUseAllCpuGroups && hasMultipleGroups;
+    // only enable CPU groups if more than one group exists
+    BOOL hasMultipleGroups = m_nGroups > 1;
+    m_enableGCCPUGroups = enableGCCPUGroups && hasMultipleGroups;
+    m_threadUseAllCpuGroups = threadUseAllCpuGroups && hasMultipleGroups;
+    m_threadAssignCpuGroups = threadAssignCpuGroups && hasMultipleGroups;
 #endif // TARGET_AMD64 || TARGET_ARM64
```

Comment:
so for the storage case they would configure it to enableGCCPUGroups but not assign to threads?

Reply:
Yea they would set `GCCpuGroup=1`, `Thread_UseAllCpuGroups=1`, `Thread_AssignCpuGroups=0`. The former two would query CPU group info in order to determine the proc count and how many threads to use for the thread pool, the last would skip assigning CPU groups to some threads as they are started.